run-test-docker-image.sh

```diff
@@ -44,16 +41,18 @@ if [ "$skip_build" = true ]; then
   echo "Skipping the build"
 else
   echo "Building image..."
+  docker system prune --force
```

Comment:
I don't think you should use prune here.  I would rather see a --no-cache to the docker build arguments.  having prune here would remove ALL of the system docker images not just the ones we care about.

Reply:
Good point. Will change.